conf/nginx.conf

```diff
@@ -0,0 +1,32 @@
+server {
+    server_name localhost;
```

Comment:
I would either leave it blank, or use `$hostname`: http://nginx.org/en/docs/http/server_names.html#miscellaneous_names

Reply:
Removed.